.github/workflows/update-deb-package-snapshots.yml

```diff
@@ -3,10 +3,15 @@ on:
   # will send emails to last editor of this cron syntax (distroless-bot)
   schedule:
     - cron:  '35 8 * * *'
+  
+permissions:
+  contents: read
+    
 jobs:
   build:
     runs-on: ubuntu-latest
     permissions:
+      contents: write
```

Comment:
I'm confused, is it overwriting what is set at line 8?

Reply:
Yes. The top level is set to `contents: read` which ensures that any new job eventually added to the workflow will have only permission to read the repo content.

The `contents: write` permission to the job is to allow the create_pull_request to run,